commands/operator-sdk/cmd/up/local.go

```diff
@@ -148,7 +148,7 @@ func upLocalAnsible() {
 	}
 
 	// start the operator
-	go ansibleOperator.Run(done, mgr, "./"+ansibleScaffold.WatchesYamlFile, time.Minute)
+	go ansibleOperator.Run(done, mgr, "./"+ansibleScaffold.WatchesYamlFile, time.Minute, true)
```

Comment:
Just commenting that we agreed in person this would be configurable in the CLI flags in a separate PR

Reply:
What's the use case for changing this at runtime vs. making it a more permanent behavior of a given operator? It seems risky to let an end user change this at runtime, vs. reserving this decision for the operator developer.